share/completions/zef.fish

```diff
@@ -0,0 +1,94 @@
+complete -c zef -n 'not __fish_is_first_token' -s v -l verbose -d 'Verbosity level'
+complete -c zef -n 'not __fish_is_first_token' -l debug -d 'Verbosity level'
+
+# Commands
+complete -c zef -n __fish_use_subcommand -a install -d 'Install specific distributions by name or path'
+complete -c zef -n __fish_use_subcommand -a uninstall -f -d 'Uninstall specific distributions'
+complete -c zef -n __fish_use_subcommand -a test -d 'Run tests on a given module\'s path'
+complete -c zef -n __fish_use_subcommand -a fetch -d 'Fetch and extract module\'s source'
+complete -c zef -n __fish_use_subcommand -a build -d 'Run the Build.pm in a given module\'s path'
+complete -c zef -n __fish_use_subcommand -a look -x -d 'Download a single module and change into its directory'
+complete -c zef -n __fish_use_subcommand -a update -d 'Update package indexes for repositories'
+complete -c zef -n __fish_use_subcommand -a upgrade -d 'Upgrade specific distributions (or all if no arguments)'
+complete -c zef -n __fish_use_subcommand -a search -d 'Show a list of possible distribution candidates for the given terms'
+complete -c zef -n __fish_use_subcommand -a info -d 'Show detailed distribution information'
+complete -c zef -n __fish_use_subcommand -a browse -d 'Browse a distribution\'s available support urls (homepage, bugtracker, source)'
+complete -c zef -n __fish_use_subcommand -a list -d 'List known available distributions, or installed distributions with `--installed`'
+complete -c zef -n __fish_use_subcommand -a depends -d 'List all direct and transitive dependencies for a given identity'
+complete -c zef -n __fish_use_subcommand -a rdepends -d 'List all distributions directly depending on a given identity'
+complete -c zef -n __fish_use_subcommand -a locate -d 'Lookup installed module information by short-name, name-path, or sha1 (with --sha1 flag)'
+complete -c zef -n __fish_use_subcommand -a smoke -d 'Run smoke testing on available modules'
+complete -c zef -n __fish_use_subcommand -a nuke -d 'Delete directory/prefix containing matching configuration path or CURLI name'
+
+# Shared options
+complete -c zef -n '__fish_seen_subcommand_from install test build' -k -a '(__fish_complete_suffix)'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l fetch
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l build
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l /build -d 'Skip the building phase'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l test
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke' -l /test -d 'Skip the testing phase'
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke depends rdepends' -l depends
+complete -c zef -n '__fish_seen_subcommand_from install upgrade smoke depends rdepends' -l /depends -d 'Do not fetch runtime dependencies'
```

Comment:
This really is `zef install -/depends`?

Reply:
Yes. Raku uses this syntax for negating an option.